tests/DiffOutputBuilder/Tests/Integration/PHPUnitPolyfill.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the GeckoPackages.
+ *
+ * (c) GeckoPackages https://github.com/GeckoPackages
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace GeckoPackages\DiffOutputBuilder\Tests;
+
+use PHPUnit\Framework\TestCase;
+use PHPUnit_Framework_TestCase;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ */
+trait PHPUnitPolyfill
+{
+    public function expectException($exception)
+    {
+        if (\method_exists(TestCase::class, 'expectException')) {
+            return parent::expectException($exception);
+        }
+
+        $this->wellYeahShipIt('expectedException', $exception);
```

Comment:
weird...
why not call `$this->setExpectedException($exception)` ?

Reply:
problem is that it will set both expected exception class and messages at the same time, not just one of the two.

Reply:
are we sure that lower branch could properly handle `expectedException` properly? I have no idea, it's internal that could be modified any time, as it's not part of public interface

Reply:
with `$this->wellYeahShipIt('expectedException', $exception);` you set only class as well ;)

Reply:
I only want to set one of the two properties, as each method only changes one. The problem is that the previous methods of PHPUnit used two properties (exception and message), while we get one property at the time so we don't know what the other one should be.